packages/tracing/src/span.ts

```diff
@@ -302,7 +311,14 @@ export class Span implements SpanInterface {
    * @inheritdoc
    */
   public getBaggage(): Baggage | undefined {
-    return this.transaction && this.transaction.metadata.baggage;
+    const existingBaggage = this.transaction && this.transaction.metadata.baggage;
+
```

Comment:
Agreed, a note would be good. I slightly rewrote this part (and the condition on when to modify baggage) in the follow-up PR (#5205). Added a note there (50e0d19ccef63b04b071c31d7e22a37db96f13a7) instead because adding it here would require it to be rewritten.